bindings/rust/s2n-tls/src/client_hello.rs

```diff
@@ -363,45 +432,8 @@ mod tests {
             0x04, 0x01, 0x05, 0x01, 0x06, 0x01, 0x02, 0x03, 0x02, 0x01, 0x00, 0x1c, 0x00, 0x02,
             0x40, 0x00,
         ];
-        let expected_fingerprint = "771,49195-49199-52393-52392-49196-49200-\
-                                    49162-49161-49171-49172-51-57-47-53-10,0-\
-                                    23-65281-10-11-35-16-5-13-28,29-23-24-25,0";
-        let expected_hash_hex = "839bbe3ed07fed922ded5aaf714d6842";
-        known_test_case(raw_client_hello, expected_fingerprint, expected_hash_hex).unwrap();
-    }
-
-    #[test]
-    fn hash_output_resizing() {
-        let client_hello = get_client_hello();
-        let hash_capacities = vec![0, MD5_HASH_SIZE, 1_000];
-        for initial_size in hash_capacities {
-            let mut hash = Vec::with_capacity(initial_size as usize);
-            client_hello
-                .fingerprint_hash(FingerprintType::JA3, &mut hash)
-                .unwrap();
-            assert_eq!(hash.len(), MD5_HASH_SIZE as usize);
-        }
-    }
-
-    #[test]
-    fn string_output_too_small() {
-        let client_hello = get_client_hello();
-        let mut fingerprint_string = String::with_capacity(0);
-        let fingerprint_err = client_hello
-            .fingerprint_string(FingerprintType::JA3, &mut fingerprint_string)
-            .unwrap_err();
-        assert_eq!(fingerprint_err.kind(), ErrorType::UsageError);
-    }
-
-    // make sure that debug doesn't panic and seems reasonable
-    #[test]
-    fn debug() {
```

Comment:
This is the only test that I deleted because it's testing that the debug struct contains the Client Hello hash, which it no longer does.